app/Activity.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Activity extends Model
+{
+    /**
+     * Mass assignable fields of Activity model
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'course_id',
+        'name',
+        'grade',
+        'porcentage'
```

Comment:
put a comma at the end of this line and write "percentage" instead of "porcentage"

Reply:
The comma is for ease of reviewing code: 

'percentage',
];